user_sync/app.py

```diff
@@ -309,9 +320,11 @@ def main():
         
         config_loader = create_config_loader(args)
         init_log(config_loader.get_logging_config())
-        
-        run_stats = user_sync.helper.JobStats("Run", divider = "=")
+
+        # add start divider, app version number, and invocation parameters to log
+        run_stats = user_sync.helper.JobStats('Run (Ver: ' + APP_VERSION + ')', divider='=')
```

Comment:
How about "User Sync version:"  No need to be terse, and much clearer for non-English speakers then "Ver".